webapp/channels/src/components/browse_channels/browse_channels.tsx

```diff
@@ -29,7 +30,14 @@ import './browse_channels.scss';
 
 const CHANNELS_CHUNK_SIZE = 50;
 const CHANNELS_PER_PAGE = 50;
-const SEARCH_TIMEOUT_MILLISECONDS = 100;
+
+var displaySettings: DisplaySettings = {
+    CustomURLSchemes: [],
+    MaxMarkdownNodes: 0,
+    SuggestionDebounceDelay: 0,
+}
+const SEARCH_TIMEOUT_MILLISECONDS: number = displaySettings.SuggestionDebounceDelay || 100;
```

Comment:
this value should be coming from a redux selector which gets this value from store which was stored with the get Config API